volume/volume_windows.go

```diff
@@ -199,3 +205,25 @@ func validateStat(fi os.FileInfo) error {
 	}
 	return nil
 }
+
+func validateAbsolute(p string) error {
```

Comment:
So, I think that after all we need a far more invasive change, and we could take the occasion to refactor a bit the volume package.

I'd like to introduce a volume.Parser interface that would expose methods to parse/validate mount specs, with a constructor taking a platform string. We would have 3 implementaions (linuxParser, windowsParser, lcowParser) that would certainly share some amount of code, instead of relying on conditional compilation and current tricks (for exemple the convertSlashes doing the exact oposite on windows and linux is SO confusing on code maintenance mode...)